WerWolv/EdiZon_CheatsConfigsAndScripts
# Add Let's Go Pikachu/Eevee cheats

This pull request:
- Adds a cheat for all wild Pokemon to have 6IVs
- Adds a cheat to the v1.0.1 updates to set wild Pokemon natures equal to the quantity of the first catch bag item
- Fixes the shiny pause cheat for the v1.0.1 updates - the original one I PR'd appeared to have offsets copied incorrectly from my notes

To test the wild Pokemon 6IV cheat:
1. Catch a Pokemon
2. Verify if has 6 IVs with a value of 31

To test the force nature cheat:
1. Ensure the first item in your catch bag has a quantity of 3 (buy/sell items at the mart)
2. Catch a Pokemon
3. Verify it has a Brave nature
